assets/app/scripts/controllers/project.js

```diff
@@ -8,19 +8,25 @@
  * Controller of the openshiftConsole
  */
 angular.module('openshiftConsole')
-  .controller('ProjectController', function ($scope, $routeParams, DataService, $filter, LabelFilter) {
+  .controller('ProjectController', function ($scope, $rootScope, $routeParams, DataService, AuthService, $filter, LabelFilter) {
+
     $scope.projectName = $routeParams.project;
     $scope.project = {};
     $scope.projectPromise = $.Deferred();
     $scope.projects = {};
 
-    DataService.get("projects", $scope.projectName, $scope, function(project) {
-      $scope.project = project;
-      $scope.projectPromise.resolve(project);
-    });
+    AuthService.withUser().then(function(user) {
+      $rootScope.user = user;
```

Comment:
it feels like this should be the responsibility of the AuthService to set user on the $rootScope.  If we are going to set it in the controller then maybe it should be on $scope not $rootScope